### PR TITLE
Finished astronomy

### PR DESCRIPTION
**Pull request by soulHawk**
Hi review my code kthx bai

### CODE REVIEW

File: AstrologyVersusAstronomy/Services/DALAstronomy.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@ public class DALAstronomy : IDALAstronomy
 		// in the Astronomy index.
 		public double GetAge(int age, string planet)
 		{
-			return 0;
+			return 100;
```
Comment (tkansa) [inline AstrologyVersusAstronomy/Services/DALAstronomy.cs:18]: change it back

File: AstrologyVersusAstronomy/Services/DALAstronomy.cs [hunk 2]
```diff
@@ -24,7 +24,7 @@ public double GetAge(int age, string planet)
 		// Be creative!
 		public List<string> GetSupplies(string planet)
 		{
-			return new List<string> { "X", "Y", "Z" };
+			return new List<string> { "A", "B", "C" };
```
Comment (tkansa) [inline AstrologyVersusAstronomy/Services/DALAstronomy.cs:27]: change it back!

**Review by tkansa — CHANGES_REQUESTED** [review]

File: AstrologyVersusAstronomy/Services/DALAstronomy.cs (hunk 1 above)
Comment (soulHawk) [reply AstrologyVersusAstronomy/Services/DALAstronomy.cs]: done

File: AstrologyVersusAstronomy/Services/DALAstronomy.cs (hunk 2 above)
Comment (soulHawk) [reply AstrologyVersusAstronomy/Services/DALAstronomy.cs]: Done

Comment (tkansa) [reply AstrologyVersusAstronomy/Services/DALAstronomy.cs]: Change it back again

**Review by tkansa — CHANGES_REQUESTED** [review]

**Review by tkansa — CHANGES_REQUESTED** [review]
Please change to ABC

**Pull request merged**